Documentation/contributing/release/stable.rst

```diff
@@ -157,12 +140,31 @@ If you intent to release a new feature release, see the
 
    #. Preview the description and then publish the release
 
-#. Announce the release in the ``#general`` channel on Slack
+#. Prepare Helm changes using the `Cilium Helm Charts Repository <https://github.com/cilium/charts/>`_:
+
+   ::
+
+      ./prepare_artifacts.sh /path/to/cilium/repository/checked/out/to/release/commit
+      git push
```

Comment:
Might worth pointing out that `git push` is not supposed to be done in `github.com/cilium/cilium` but to `github.com/cilium/helm`, right?

Reply:
`./prepare_artifacts` won't work if you followed the instructions wrong right? I thought it was clear by linking directly to the charts repository :<